content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-1-0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: v1.1.0 Armory Operator
+toc_hide: true
+---
+
+## 08/11/2020 Release Notes
+
+## Known Issues
+No known issues.
```

Comment:
Ingress support is a known issue at least because of the missing permissions in `role.yaml` manifest. So at least a note like "if you want to use ingress support, please add this permission to the role that operator uses".
Even after adding the permission ingress may not work, we need to document in which case it works and in which case it doesn't.

Reply:
Aren't we going to provide it in the manifest? I guess it would be a patch release 🤷